src/lib.rs

```diff
@@ -169,7 +169,16 @@ where
     }
 
     ///
-    /// Sets pixel colors in given rectangle bounds.
+    /// Sets pixel colors in a rectangular region.
+    ///
+    /// The color values from the `colors` iterator will be drawn to the given region starting
+    /// at the top left corner and continuing, row first, to the bottom right corner. No bounds
+    /// checking is performed on the `colors` iterator and drawing will wrap around if the
+    /// iterator returns more color values than the number of pixels in the given region.
+    ///
+    /// This is a low level function, which isn't intended to be used in regular user code.
+    /// Consider using the [`fill_contiguous`](https://docs.rs/embedded-graphics/latest/embedded_graphics/draw_target/trait.DrawTarget.html#method.fill_contiguous)
+    /// function from the `embedded-graphics` crate as an instead.
```

Comment:
Something is missing in this sentence:
```suggestion
    /// function from the `embedded-graphics` crate as an alternative instead.
```

Reply:
Oops! I don't know how that slipped through. Thank you for all the patience and help!

> It's sometimes hard to know what is missing if you are already very familiar with the create.

And coming from the outside, it's difficult to tell how something is meant to be used without clear instructions - That's why I was so off with `set_pixels` and why I wanted to add some examples :)